src/main/controllers/helpers/TribunalOrderOrRequestHelper.ts

```diff
@@ -316,27 +276,41 @@ export const getNotificationResponses = async (
   translations: AnyRecord,
   req: AppRequest
 ): Promise<any> => {
+  const nonAdminRespondCollection = sendNotificationType?.respondCollection
+    ? sendNotificationType.respondCollection
+    : [];
+  const adminResponseCollection = sendNotificationType?.respondNotificationTypeCollection
+    ? sendNotificationType?.respondNotificationTypeCollection
+    : [];
   const allResponses: any[] = [];
-  const respondCollection = sendNotificationType.respondCollection;
-  if (respondCollection?.length) {
-    await addClaimantRespondentResponses(respondCollection, req, translations, allResponses);
-  }
-
-  const adminResponseCollection = sendNotificationType.respondNotificationTypeCollection;
-  if (adminResponseCollection?.length) {
-    await addTribunalResponses(adminResponseCollection, req, allResponses, translations);
+  const combinedResponses = [...nonAdminRespondCollection, ...adminResponseCollection];
+  if (combinedResponses?.length) {
```

Comment:
minor: no need for `?` on combinedResponses

minor: could inverse this condition and return early

Reply:
Done